lnwallet/chanfunding/assembler.go

```diff
@@ -59,6 +59,21 @@ type Request struct {
 	// the funding output.
 	RemoteAmt btcutil.Amount
 
+	// FundUpToMaxAmt should be set to a non-zero amount if the channel
+	// funding should try to add as many funds to LocalAmt as possible
+	// until at most that amount is reached.
+	FundUpToMaxAmt btcutil.Amount
+
+	// MinFundAmt should be set iff the FundUpToMaxAmt field is set. It
+	// either carries the configured minimum channel capacity or, if an
+	// initial remote balance is specified, enough to cover the initial
+	// remote balance.
+	MinFundAmt btcutil.Amount
+
+	// RemoteChanReserve is the channel reserve we required for the remote
+	// peer.
+	RemoteChanReserve btcutil.Amount
```

Comment:
We need it because we cannot do the following check in the 'parseChannelOpenReq' but have to wait till the coin selection has succeeded: https://github.com/lightningnetwork/lnd/pull/6903/files#diff-674c79ba02b6e9c45a994388392689814f3538800351b2fd43fb3ade21e1effbR2013

The additional check is here: https://github.com/lightningnetwork/lnd/pull/6903/files#diff-ba25a5144f3090a192b11b77b5cbd5b63d1d33cfe61dd1e29cb15dabb27261e9R315

Reply:
Oh now I see it. Yes, `VerifyConstraints` tests exactly this. I will remove `RemoteChanReserve` and the check I added in the wallet.

Reply:
f